app/src/main/java/com/practicum/playlistmaker/mediateka/ui/FavoriteState.kt

```diff
@@ -0,0 +1,13 @@
+package com.practicum.playlistmaker.mediateka.ui
+
+import com.practicum.playlistmaker.search.domain.Track
+
+
+sealed interface FavoriteState {
+
+    data class Content(
+        val tracks: List<Track>
+    ) : FavoriteState
```

Comment:
```suggestion
    data class Content(val tracks: List<Track>) : FavoriteState
```

Если параметр 1 - то он пишется в 1 строку, если 2 и более - каждый новый параметр с новой строки